src/libmongoc/src/mongoc/mongoc-opts.c

```diff
@@ -1563,6 +1652,15 @@ _mongoc_change_stream_opts_parse (
             return false;
          }
       }
+      else if (!strcmp (bson_iter_key (&iter), "comment")) {
+         if (!_mongoc_convert_bson_value_t (
+               client,
+               &iter,
+               &mongoc_change_stream_opts->comment,
+               error)) {
+            return false;
+         }
+      }
```

Comment:
No a comment on this PR in particular, but the repetition of this pattern over and over and over again looks like a good opportunity to bring out the blunt hammer of C preprocessor macros.

Reply:
Oh, these files are all generated from templates in `generate-opts.py`. Did you mean that this pattern is used throughout libmongoc or just the generated code? If the latter, I don't think it's worth creating a macro since this files isn't intended to be manually modified anyway.